pkg/gomod/bytes/parser_test.go

```diff
@@ -0,0 +1,26 @@
+package bytes
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+)
+
+func TestContentParser_ModuleName(t *testing.T) {
+
+	moduleName := "github.com/gomods/athens"
```

Comment:
try also with some invalid names, run data table test to check it passes with simple names, names with slashes, dashes dots ... anything you can pass to URL
and does not pass with some invalid names like '?' etc. just a few cases so we're safe.

Reply:
I added those in https://github.com/adelowo/athens/blob/cc43aa74b6fb45068a3fea07105e4849c9393db5/pkg/gomod/parser_test.go#L25 ... Would just improve the test cases there

Reply:
cool, sorry missed that. i'm just skimming, i will give a thorough review in few hours

Reply:
Some improvements in https://github.com/gomods/athens/pull/71/commits/1502473729053f0980556ff9de1cf5298b8414c8